integration/src/main/java/scripts/common/ApiHelpers.java

```diff
@@ -0,0 +1,43 @@
+package scripts.common;
+
+import bio.terra.datarepo.api.JobsApi;
+import bio.terra.datarepo.api.SnapshotsApi;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.client.ApiException;
+import bio.terra.datarepo.model.JobModel;
+import bio.terra.datarepo.model.SnapshotRequestModel;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+public class ApiHelpers {
```

Comment:
Since each `ApiClient` is specific to a service, I would put this code inside `DatarepoClient` (vs a more generally named `ApiHelpers`)

Reply:
if I move createSnapshot and deleteSnapshot to DatarepoClient I think it makes the code awkwardly organized, since DatarepoClient will have the create and delete functions while SnapshotsApi will still have addSnapshotPolicyMember and other API functions

Reply:
That's true, they don't really belong in `DatarepoClient`. Could you subclass `SnapshotsApi` and add the methods to that? Something like

```
public class SnapshotsApi extends bio.terra.datarepo.api.SnapshotsApi {
  public SnapshotsApi(DatarepoClient apiClient) {
    super(apiClient);
  }

  public DatarepoClient getApiClient() {
    return (DatarepoClient) super.getApiClient();
  }

  public void synchronousDeleteSnapshot(UUID snapshotId) throws Exception {
    String deleteJobId = deleteSnapshot(snapshotId).getId();
    getApiClient().waitForJobSuccess(deleteJobId);
  }

  public UUID synchronousCreateSnapshot(SnapshotRequestModel request) throws Exception {
    String createJobId = createSnapshot(request).getId();
    Map<Object, Object> result = getApiClient().waitForJobSuccess(createJobId);
    return UUID.fromString((String) result.get("id"));
  }
}
```

And in `DatarepoClient`:
```
  public SnapshotsApi createSnapshotsApi() {
    return new SnapshotsApi(this);
  }
```

Reply:
(inheritance makes this easier to write but encapsulation may be a better approach)